yugabyte/yugabyte-db
# [docdb] Tool to flush all tablets on a tablet server #2785

solve issue #2785 and fix the tablet server failure of flushing system.transactions

Test Plan:
1. Create a single-node cluster by `./bin/yb-ctl create`.
2. Create new database in YSQL shell and load sample data from `sample/`. (similar to https://docs.yugabyte.com/latest/quick-start/explore-ysql/#linux)
3. For the test of `flush_tablet` and `compact_tablet`, pick a tablet id by `./build/latest/bin/yb-ts-cli list_tablets`.
4. Test the command as followed:
`./build/latest/bin/yb-ts-cli flush_tablet <tablet_id>`
`./build/latest/bin/yb-ts-cli flush_all_tablets`
`./build/latest/bin/yb-ts-cli compact_tablet <tablet_id>`
`./build/latest/bin/yb-ts-cli compact_all_tablets`
5. Check whether there are new sst files generated in `yugabyte-data/node-1/disk-1/yb-data/tserver/data/rocksdb/table-<table_id>/tablet-<tablet_id>/`
